src/hidpp20.c

```diff
@@ -2627,7 +2627,7 @@ hidpp20_onboard_profiles_initialize(struct hidpp20_device *device,
 						  profiles->sector_size,
 						  data);
 
-	if (rc && device->quirk != HIDPP20_QUIRK_G305) {
+	if (rc && device->quirk == HIDPP20_QUIRK_G305) {
```

Comment:
@FFY00 - typo, right? this patch is correct?

Reply:
Yes!